src/webapp/models.py

```diff
@@ -146,6 +148,41 @@ def get_contacts_data(self) -> ContactsData:
 
         return self.contacts_data.data
 
+    def get_comanage_data(self) -> ContactsData:
```

Comment:
Like with get_contact_db_data, there should be a config knob to skip this and return an empty ContactsData.

Reply:
Updated - now if any of the `CILOGON_LDAP_*` vars are None, an empty ContactsData is returned.

(`CILOGON_LDAP_URL` & `_USER` have their the expected values as defaults; though `_PASSFILE` defaults to None)